internal/github/profile_test.go

```
@@ -0,0 +1,333 @@
+			errorAssertion: assert.Error,
+		},
+		{
+			configURL:      "github.com/chinmina/chinmina-bridge/docs/invalid-profile.yaml",
+			expectedConfig: github.ProfileConfig{},
+			errorAssertion: assert.Error,
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.configURL, func(t *testing.T) {
+			updatedURL, err := url.Parse(tc.configURL)
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)